src/main/java/frc/team4909/robot/subsystems/leds/LEDs.java

```diff
@@ -0,0 +1,39 @@
+package frc.team4909.robot.subsystems.leds;
+
+import edu.wpi.first.wpilibj.Spark;
+
+public class LEDs{
+    public Spark LEDController;
+
+    public LEDs() {
+        LEDController = new Spark(0);
+    }
+
+    public void setRed(){
+        LEDController.set(0.59);
+    }
+
+    public void setGreen(){
+        LEDController.set(0.77);
+    }
+
+    public void setBlue(){
+        LEDController.set(0.85);
+    }
+
+    public void setBlack(){
+        LEDController.set(0.99);
+    }
+
+    public void setRedBlink(){
+        LEDController.set(-0.11);
+    }
+
+    public void setDisabled(){
+        LEDController.set(-0.71);
```

Comment:
All other settings are clear what they do from the name of their enclosing function. In this case, disabled doesn't indicate anything about a color. Negative seems to imply blinking, but blinking what? A comment to describe here what it's doing (and why) would be beneficial.